include/jwt-cpp/jwt.h

```diff
@@ -861,19 +860,19 @@ namespace jwt {
 			auto decoded_modulus = base::decode<alphabet::base64url>(base::pad<alphabet::base64url>(modulus));
 			auto decoded_exponent = base::decode<alphabet::base64url>(base::pad<alphabet::base64url>(exponent));
 
-			std::unique_ptr<BIGNUM, decltype(&BN_free)> n(
-				BN_bin2bn(reinterpret_cast<const unsigned char*>(decoded_modulus.data()),
-						  static_cast<int>(decoded_modulus.size()), nullptr),
-				BN_free);
+			BIGNUM* n = BN_bin2bn(reinterpret_cast<const unsigned char*>(decoded_modulus.data()),
+						  static_cast<int>(decoded_modulus.size()), nullptr);
 
-			std::unique_ptr<BIGNUM, decltype(&BN_free)> e(
-				BN_bin2bn(reinterpret_cast<const unsigned char*>(decoded_exponent.data()),
-						  static_cast<int>(decoded_exponent.size()), nullptr),
-				BN_free);
+			BIGNUM* e = BN_bin2bn(reinterpret_cast<const unsigned char*>(decoded_exponent.data()),
```

Comment:
Why did you remove the unique ptr? According to https://www.openssl.org/docs/man3.1/man3/BN_bin2bn.html it needs to be freed 🤔